aquadrone_pneumatics/launch/pneumatics.launch

```diff
@@ -0,0 +1,3 @@
+<launch>
+    <node name="pneumatics" pkg="aquadrone_pneumatics" type="pneumatics_node.py" output="screen"/>
```

Comment:
The node file should take a parameter called real. If real == True, it should launch the real pneumatics node, otherwise it should launch the simulated pneumatics node. Check out this file for an example: https://github.com/Waterloo-Aquadrone/aquadrone-core/blob/dev/thruster_control/launch/control.launch

Reply:
e878a21